### ISSUE TITLE
Fresh install stuck on "Before we start..." consent form, "Invalid Path" response

### ISSUE THREAD

**Issue by LivingWithHippos** [Countly/countly-server#5824]
### Expected Behavior

After a fresh install, selecting yes or no to the consent form for the telemetry and the newsletter should bring me to the homepage

### Current Behavior

The page gets reloaded with the consent form, choosing yes or no won't do anything. The newsletter form does not reappear, suggesting that gets accepted. 

> We utilize Countly to understand user interactions and collect feedback, helping us enhance our product continuously. However, your privacy remains our priority. This analysis is done on the server level, so we won't see or collect any individual details or any data you record. The data is reported back only to our dedicated Countly server based in Europe. Please note, you can change your mind at any time in the settings.

### Possible Solution

Aside from the "correct" fix would it be possible to bypass this form?

### Steps to Reproduce (for bugs)

1. Fresh install of latest countly
2. Create a new user
3. Create a new app or demo app
4. Answer the consent forms

### More Description

Tried using the demo app or creating one from zero, and choosing yes or no to the consent form, without success

On the page `https://my.domain/dashboard#/123_my_app_id/not-responded-consent` these calls

```
https://my.domain/i/configs?configs={"api":{"domain":"https://my.domain"}}&app_id=123_my_app_id
https://my.domain/o/configs?app_id=123_my_app_id
https://my.domain/o/themes?app_id=123_my_app_id
```

all result in a 400 response with

```json
{"result":"Invalid path"}
```

all other calls including to `o?app_id=123_my_app_id`  `/i/cms/save_entries`  and to `cms.count.ly` work

Reloading the containers allow me to log in (so the user and app do get saved) but directly go to the consent page

Removing volumes and containers and recreating everything reproduces the issue

### Your Environment

Note: I had an older installation (countly 20) with these changes and it worked

* Operating System and version:

Ubuntu 22

* Any changes you have made to your instance: 

- removed the exposed ports from nginx and used caddy to pass everything to it and provide https
- added container names to the compose file
- edited the nginx file to use the new container names

<details><summary>docker compose</summary>

```yaml
services:
  countly-api-2:
    image: "countly/api:latest"
    container_name: countly_api_2
    environment:
      - COUNTLY_PLUGINS=${COUNTLY_PLUGINS}
      # - COUNTLY_CONFIG__MONGODB_HOST=mongodb2
      - COUNTLY_CONFIG_API_API_WORKERS=4 # CPU core count
      - COUNTLY_CONFIG__FILESTORAGE="gridfs"
      - COUNTLY_CONFIG__MONGODB="mongodb://${MONGO_USER}:${MONGO_PASSWORD}@${MONGODB_HOST}/countly?maxPoolSize=500"
      - NODE_OPTIONS="--max-old-space-size=2048"
      - COUNTLY_CONFIG_HOSTNAME=${COUNTLY_HOSTNAME}
    deploy:
      restart_policy:
        condition: on-failure
        max_attempts: 3
    depends_on:
      - mongodb2
    networks:
      - backend

  countly-frontend-2:
    image: "countly/frontend:latest"
    container_name: countly_frontend_2
    environment:
      - COUNTLY_PLUGINS=${COUNTLY_PLUGINS}
      # - COUNTLY_CONFIG__MONGODB_HOST=mongodb2
      - NODE_OPTIONS="--max-old-space-size=2048"
      - COUNTLY_CONFIG__MONGODB="mongodb://${MONGO_USER}:${MONGO_PASSWORD}@${MONGODB_HOST}/countly?maxPoolSize=500"
      - COUNTLY_CONFIG_HOSTNAME=${COUNTLY_HOSTNAME}
    depends_on:
      - mongodb2
    deploy:
      # There is usually no need in multiple frontends, so throttling down resources for it
      mode: global
      resources:
        limits:
          cpus: "0.5"
      restart_policy:
        condition: on-failure
        max_attempts: 3
    networks:
      - backend

  nginx2:
    image: "bitnami/nginx"
    # ports:
    #  - "80:8080"'
    container_name: countly_proxy_2
    volumes:
      - "./nginx.server.conf:/opt/bitnami/nginx/conf/server_blocks/countly.conf:ro"
    networks:
      - backend
      - caddy
    depends_on:
      - countly-api-2
      - countly-frontend-2

  mongodb2:
    image: "bitnami/mongodb:8.0"
    container_name: countly_mongodb_2
    environment:
      - MONGODB_ROOT_PASSWORD=${MONGO_4_INITDB_ROOT_PASSWORD}
    volumes:
      - "mongodb_data_countly:/bitnami"
      - ./mongo_init.js:/docker-entrypoint-initdb.d/mongo-init.js:ro
    networks:
      - backend

volumes:
  mongodb_data_countly:

networks:
  backend:
    name: countly_backend
    driver: bridge
  caddy:
    external: true
```

</details>

<details><summary>nginx conf</summary>

```bash
server {
	listen   8080;
	listen   [::]:8080 ipv6only=on;
	server_name  localhost;

	access_log  off;

	location = /i {
        if ($http_content_type = "text/ping") {
             return 404;
        }
		proxy_pass http://countly_api_2:3001;
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
		proxy_set_header X-Real-IP $remote_addr;
	}

	location ^~ /i/ {
        if ($http_content_type = "text/ping") {
             return 404;
        }
		proxy_pass http://countly_api_2:3001;
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
		proxy_set_header X-Real-IP $remote_addr;
	}

	location = /o {
        if ($http_content_type = "text/ping") {
             return 404;
        }
		proxy_pass http://countly_api_2:3001;
	}

	location ^~ /o/ {
        if ($http_content_type = "text/ping") {
             return 404;
        }
		proxy_pass http://countly_api_2:3001;
	}

	location / {
        if ($http_content_type = "text/ping") {
             return 404;
        }
		proxy_pass http://countly_frontend_2:6001;
		proxy_set_header Host $http_host;
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
		proxy_set_header X-Real-IP $remote_addr;
	}
}
```

</details>

**Comment by ar2rsawseen** [comment]
Hello,
I think this has been fixed on cms side, but if it repeats, let us know

**Comment by LivingWithHippos** [comment]
Still have this behaviour on `25.03.3`, maybe I should wait for another release?

**Comment by widatama** [comment]
Hello @LivingWithHippos 
Thank you for the logs. Glad to know that the script helped.
From what I can gather so far, the frontend is trying to update this endpoint https://my.domain/i/configs?configs but it get's an error, so it can't continue to the dashboard page.
Now we'll investigate the error in that endpoint.
From the log, it looks like there's nothing wrong with the request itself, so it's possible that the request did not reach countly server correctly.
Can we do some more tests on your side?

First, try to update setting from the dashboard and see if there's any error with the update.
For example, you can update setting in this page.

![Image](https://github.com/user-attachments/assets/fa48ba63-3079-4745-b4ef-a10f352f86d4)

This is to check if the request from browser can reach countly server correctly.

Second, try to update setting from inside the api container.
You can do this with curl like 
`curl 'http://localhost:3001/i/configs?configs=%7B%22frontend%22%3A%7B%22countly_tracking%22%3Afalse%7D%7D&api_key=api_key_value'`
This is to check if there's something wrong with the server endpoint.

**Comment by LivingWithHippos** [comment]
I don´t have settings under management, do I need to install it as a plugin?

**Comment by widatama** [comment]
Wait, is your user not global admin? This is the user that you create from setup page right?

**Comment by widatama** [comment]
I think the plugins that you listed should be enough for your purpose.

As for the settings menu, it should appear once the plugin `plugins` is active.
Could you please try rebuilding the container and see if the menu appears.

In the meantime we haven't been able to recreate your setup. So if possible could you provide the necessary configs and environment that is needed to run the setup.
